arviz/data/io_dict.py

```diff
@@ -44,7 +49,7 @@ def posterior_to_xarray(self):
         if "log_likelihood" in data:
             warnings.warn(
                 "log_likelihood found in posterior."
-                " For stats functions log_likelihood needs to be in sample_stats.",
+                " For stats functions log_likelihood needs to be in log_likelihoods.",
```

Comment:
Strange error message, I might be confused if I read that.

Reply:
using `be in 'log_likelihoods' group` would be clearer?

Reply:
Or whatever group name we agree on eventually